packages/esm-patient-medications-app/src/add-drug-order/drug-search/drug-search.component.tsx

```diff
@@ -8,13 +8,14 @@ import styles from './order-basket-search.scss';
 
 export interface DrugSearchProps {
   openOrderForm: (searchResult: DrugOrderBasketItem) => void;
+  debounceTime?: number;
 }
 
-export default function DrugSearch({ openOrderForm }: DrugSearchProps) {
+export default function DrugSearch({ openOrderForm, debounceTime = 300 }: DrugSearchProps) {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const [searchTerm, setSearchTerm] = useState('');
-  const debouncedSearchTerm = useDebounce(searchTerm);
+  const debouncedSearchTerm = useDebounce(searchTerm, debounceTime);
```

Comment:
[useDebounce](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-react-utils/src/useDebounce.ts#L32) already defaults to a timeout of 300ms. Maybe a better approach here would be to make the debounce value a configurable property in the medications app's config schema and then using whatever value is provided, or the default otherwise.

Reply:
@denniskigen Thanks so much for the quick review let me try the fix.